centinel/client.py

```diff
@@ -163,7 +199,7 @@ def run(self, data_dir=None):
                     logging.info("tcpdump stopped.")
                     try:
                         pcap_file_name = "pcap_%s-%s.pcap.bz2" % (name, 
-                            datetime.now().isoformat())
+                            exp_start_time)
```

Comment:
Adding more indent to match where "name," starts will make the line longer than 78 chars and violates PEP8.